backend/Watchdog.Core/Watchdog.Core.BLL/Services/MemberService.cs

```diff
@@ -1,30 +1,94 @@
 ﻿using AutoMapper;
 using Microsoft.EntityFrameworkCore;
+using SendGrid;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Watchdog.Core.BLL.Models;
 using Watchdog.Core.BLL.Services.Abstract;
-using Watchdog.Core.Common.DTO.Member;
+using Watchdog.Core.Common.DTO.Members;
 using Watchdog.Core.DAL.Context;
 using Watchdog.Core.DAL.Entities;
 
 namespace Watchdog.Core.BLL.Services
 {
+
     public class MemberService : BaseService, IMemberService
     {
-        public MemberService(WatchdogCoreContext context, IMapper mapper) : base(context, mapper)
+        private readonly IEmailSendService _emailSendService;
+
+        public MemberService(WatchdogCoreContext context, IMapper mapper, IEmailSendService emailSendService) : base(context, mapper)
         {
+            _emailSendService = emailSendService;
         }
 
-        public async Task<ICollection<MemberDto>> GetMembersByOrganizationIdAsync(int id)
+        public async Task<Response> InviteMemberAsync(int id)
```

Comment:
Response is not a good name of class

Reply:
It is not my class. This is class that encapsulates response from SendGrid
![image](https://user-images.githubusercontent.com/75535425/129493613-d4fa55ab-931a-42ad-9ebc-71a66be97668.png)